clusterloader2/testing/density/config.yaml

```diff
@@ -8,6 +8,7 @@
 {{$NODES_PER_NAMESPACE := DefaultParam .NODES_PER_NAMESPACE 100}}
 {{$PODS_PER_NODE := DefaultParam .PODS_PER_NODE 30}}
 {{$DENSITY_TEST_THROUGHPUT := DefaultParam .DENSITY_TEST_THROUGHPUT 20}}
+{{$SCHEDULER_THROUGHPUT_THRESHOLD := DefaultParam .CL2_SCHEDULER_THROUGHPUT_THRESHOLD -1}}
```

Comment:
Why not set it to 0 as default?
Throughput will always be >= 0, so then you won't need if below.

Reply:
Done.